gateway/src/main/java/com/lyft/data/gateway/handler/QueryIdCachingProxyHandler.java

```diff
@@ -262,7 +299,7 @@ protected void postConnectionHook(
     queryDetail.setSource(request.getHeader(SOURCE_HEADER));
     String queryText = CharStreams.toString(request.getReader());
     queryDetail.setQueryText(
-        queryText.length() > 400 ? queryText.substring(0, 400) + "..." : queryText);
+        queryText.length() > 400 ? queryText.substring(0, 200) + "..." : queryText);
```

Comment:
Mismatch between length check of 400 and substring of 200.

Reply:
good catch :)
I wanted to reduce this query content so that page could load sooner (with lesser data). I will update this to be consistent. Thanks.